gitCoreApi/src/main/java/com/virtuslab/gitcore/api/GitCoreBranchTrackingStatus.java

```diff
@@ -1,11 +1,9 @@
-package com.virtuslab.gitcore.impl.jgit;
+package com.virtuslab.gitcore.api;
 
 import lombok.Data;
 
-import com.virtuslab.gitcore.api.IGitCoreBranchTrackingStatus;
-
 @Data(staticConstructor = "of")
-public class GitCoreBranchTrackingStatus implements IGitCoreBranchTrackingStatus {
```

Comment:
I don't think interface/impl split was justified for a simple and rather implementation-independent stuff like this